frontend/packages/dev-console/src/components/monitoring/alerts/__tests__/rules-mocks.ts

```diff
@@ -0,0 +1,271 @@
+import { PrometheusRulesResponse } from '@console/internal/components/monitoring/types';
```

Comment:
Let's reduce this down to just what we need for the tests so that it's easier to keep in sync with future data format changes.

Reply:
Done.